apps/showcase/src/components/showcase/localization/localization-pres.config.ts

```diff
@@ -0,0 +1,12 @@
+import type {Configuration} from '@o3r/core';
+import {computeItemIdentifier} from '@o3r/core';
+
+export interface LocalizationPresConfig extends Configuration {
+  datePickerCustomKey: string;
```

Comment:
Could we have a dedicated component for component replacement instead of using the localization one ?

Reply:
I was thinking to create a new chapter for this, but I was thinking to keep it in another feature which will include the update of the doc for component replacement, because the things are a bit simplified with the standalone components (which are quite recent). 
And as a quick win I put this component here to have an example until we decide to have a dedicated chapter in the showcase.